### PR TITLE
[KIALI-646] Keep only ReplicaSet on the Template

### PR DESCRIPTION
**Pull request by gbaufake**
After merging #6, we had deployment and replicaset on the template.
Removing and leaving only replicaSet

### CODE REVIEW

**Comment by cfcosta** [comment]
I tried this PR locally, and it is not showing anything on the Graph, is that expected?

/cc @mwringe

**Comment by mattmahoneyrh** [comment]
@cfcosta @mwringe quick test: when I exposed the services and then navigated to the routes, yes there were services and connections between the services on the Service Graph.

**Comment by cfcosta** [comment]
I tested it again here, if I expose the services and query them directly they do appear on the listing. Unless I'm misunderstanding it, the traffic-generator should be generating the data directly, no?

/cc @gbaufake @mattmahoneyrh

File: test-service/deploy/openshift/test-app-template.yaml [hunk 1]
```diff
@@ -26,51 +26,32 @@ objects:
...
   spec:
     replicas: 1
     selector:
       matchLabels:
-        kiali-test: test-service-${SERVICE_NAME}
+        kiali: test-service-${SERVICE_NAME}-${SERVICE_VERSION}
     template:
       metadata:
         labels:
-          kiali-test: test-service-${SERVICE_NAME}
```
[the remark this answers is not in the document]
Comment (gbaufake) [reply test-service/deploy/openshift/test-app-template.yaml]: Ok I'll update

**Comment by mwringe** [comment]
@cfcosta The traffic generator should be generating traffic, if its configured and setup properly. Maybe @gbaufake can take a look at it?

**Comment by gbaufake** [comment]
@mwringe  looking into it

**Comment by gbaufake** [comment]
@cfcosta have you got test mesh running?

I tried to curl it manually and I didn't get the graph like the #4.

What would it be the curl request for services with different pods? (considering that we going to have just one service with multiples pods)

**Pull request merged**